### PR TITLE
Improve output of buildInfo command

### PR DESCRIPTION
**Pull request by GinGin3203**
PR for #197 and #198.

Although I'm not completely sure about obtaining the value of architecture bits, but adding another flag to `buildInfo.Settings` seems a bit cleaner than parsing `GOARCH` (especially if you consider that, for example, `amd64p32` is a valid 32-bit compilation target).

### CODE REVIEW

**Comment by AlekSi** [comment]
> Although I'm not completely sure about obtaining the value of architecture bits

We currently use `strconv.IntSize` in `MsgHostInfo`.

> but adding another flag to buildInfo.Settings seems a bit cleaner

`buildInfo.Settings` is populated by `go build`, we can't easily change that.

> amd64p32 is a valid 32-bit compilation target

Support for amd64p32 was removed in 1.14 together with NaCl: https://go.dev/doc/go1.14#nacl Currently, there are no plans to support weird architectures like this.

**Comment by AlekSi** [comment]
See also https://github.com/FerretDB/FerretDB/pull/205

File: internal/util/version/version.go [hunk 1]
```diff
@@ -27,9 +27,10 @@ import (
 var version string
 
 type Info struct {
-	Version string
-	Commit  string
-	Dirty   bool
+	Version      string
+	Commit       string
+	Dirty        bool
+	Architecture int32 // either equals to 32 or 64
```
Comment (AlekSi) [inline internal/util/version/version.go:33]: That's unused now, right?

Comment (GinGin3203) [reply internal/util/version/version.go:33]: Yup, gonna fix it right away

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Thank you!

**Pull request merged**